src/test/resources/features/ErrorMessages.feature

```diff
@@ -135,13 +135,57 @@ Feature: Book a Visit Error messages
     Then I see "No main contact selected" on select main contact page
     And I select the main contact option
     And click on continue button
+    Then I see "No answer selected" error message on main contact page
+    And I choose UK phone number option
+    And click on continue button
+    Then I see "Enter a phone number" select main contact page
+    And I sign out of the service
```

Comment:
Just thinking.... if we have a lot of tests that just stop part-way through the booking journey then we will have a lot of pending 'reservations' that will not disappear for ~10 mins - so potentially making tests flakey. More so the more of these abandoned journeys we have.

Would any of these ideas be worth considering:
* surfacing the application reference (e.g. hidden attribute on the page) so you could use testing helper to 'clean up' and get that application removed? (cc @dhirajshettymoj  - don't know if this would work!)
* in a case like this, after checking the errors, continue to book and then cancel the visit as a clean up step
* perhaps have fewer tests but each one tests lots of validation errors on each page, so fewer abandoned journeys. Might get unwieldy though 🤷‍♂️